### DISCUSSION TITLE
Missing Vim motions

### DISCUSSION THREAD

**Discussion by abhiiguptaa** [cpeditor/cpeditor#1267] [Ideas]
Please add the vim motion inside this editor

**Comment by abhiiguptaa** [comment]
I like your editor very and I also wanted to use your editor but I can't use it because it does not support the vim motions or emulations like h, j, k, l binding to move the cursor up and down and other useful key. Everything is perfect except this. Please add this feature in the upcoming updates because I wanted to use the editor which is light weight and easy to use. Thanks

**Comment by ouuan** [comment]
It is tracked at #220.
